gravitee-apim-console-webui/src/management/api/component/api-import-file-picker/api-import-file-picker.component.html

```diff
@@ -15,7 +15,13 @@
     limitations under the License.
 
 -->
-<gio-form-file-picker [accept]="accept" [ngModel]="filePickerValue" (ngModelChange)="onImportFile($event)">
+<gio-form-file-picker
+  class="file-picker"
+  [class.file-picker--filled]="filePickerValue?.length > 0"
```

Comment:
Just because I see it there but we could have used a [formControl] and listened to the changes with rxjs to be more in line with Angular best practice.  Just a thought